packages/synapse-react-client/src/utils/AppUtils/session/ApplicationSessionManager.tsx

```diff
@@ -91,7 +91,7 @@ export function ApplicationSessionManager(
       initAnonymousUserState()
       return
     }
-    if (maxAge) {
+    if (maxAge !== undefined && !!token) {
```

Comment:
To be safe, let's only use the maxAge parameter to force logout iff maxAge is defined and we are logged in